src/syscheckd/create_db.c

```diff
@@ -53,8 +53,7 @@ static const char *FIM_EVENT_MODE[] = {
 void update_wildcards_config();
 
 
-void fim_generate_delete_event(fdb_t *fim_sql,
-                               fim_entry *entry,
+void fim_generate_delete_event(char *entry,
```

Comment:
Let's use the name `entry` only on `fim_entry` types
```suggestion
void fim_generate_delete_event(char *file_path,
```